{"name": "Изменяем данные пользователя неавторизованным пользователем - name", "status": "failed", "statusDetails": {"message": "AssertionError: assert (403 == 200)\n +  where 403 = <Response [403]>.status_code", "trace": "self = <test_change_data.TestChange object at 0x105168d90>\n\n    @allure.title('Изменяем данные пользователя неавторизованным пользователем - name')\n    def test_change_not_auth_user_data_name(self):\n        user = generate_user_data()\n        created_user_response = requests.post(f'{Urls.burger_url}{Urls.register_user}', json=user)  # Создаём нового пользователя\n        new_name = generated_login()\n        new_data = {\n            'email': user['email'],\n            'name': new_name\n        }\n        user_data_change = requests.patch(f'{Urls.burger_url}{Urls.user_info}', json=new_data)  # меняем данные пользователя\n        user_info = user_data_change.json()  # получаем информацию об ошибке\n    \n>       assert created_user_response.status_code == 200 and created_user_response.json()['success']\nE       assert (403 == 200)\nE        +  where 403 = <Response [403]>.status_code\n\ntests/test_change_data.py:80: AssertionError"}, "start": 1757231238810, "stop": 1757231239481, "uuid": "40f8f822-0feb-46ca-b1f5-8b2b91840d8f", "historyId": "2b68a92b52599e136ea0d355cb885474", "testCaseId": "2b68a92b52599e136ea0d355cb885474", "fullName": "tests.test_change_data.TestChange#test_change_not_auth_user_data_name", "labels": [{"name": "suite", "value": "Тестируем изменение данных пользователя"}, {"name": "parentSuite", "value": "tests"}, {"name": "subSuite", "value": "TestChange"}, {"name": "host", "value": "MacBook-Air-Valeria.local"}, {"name": "thread", "value": "92540-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_change_data"}], "titlePath": ["tests", "test_change_data.py", "TestChange"]}